### ISSUE TITLE
Constant Console Log Messages

### ISSUE THREAD

**Issue by razorbear3** [Hammerspoon/hammerspoon#2806]
I am getting the following messages filling up my console:

```
2021-04-05 16:10:24: targetApp: nil
2021-04-05 16:10:24: keyDelay: 0
2021-04-05 16:10:24: targetApp: nil
2021-04-05 16:10:24: keyDelay: 0
2021-04-05 16:10:24: targetApp: nil
2021-04-05 16:10:24: keyDelay: 0
2021-04-05 16:10:24: targetApp: nil
2021-04-05 16:10:24: keyDelay: 0
2021-04-05 16:10:24: targetApp: nil
2021-04-05 16:10:24: keyDelay: 0
```

The messages appear in the console when the following command is executed:
`hs.eventtap.keyStroke({},hs.keycodes.map.down, 0)`

These messages don't cause any functional issues for me, but I wanted to point them out, in case there is a stray print statement somewhere. 

If I execute the above command straight in the console it exhibits this behavior:
```
> hs.eventtap.keyStroke({},hs.keycodes.map.down, 0)
2021-04-05 17:00:14: targetApp: nil
2021-04-05 17:00:14: keyDelay: 0
```

**Comment by latenitefilms** [comment]
@cmsj has already removed these debugging messages, so it will be fixed in the next release.

**Comment by cmsj** [comment]
Indeed, this should now be fixed in 0.9.87 :)
